spec/button_spec.rb

```diff
@@ -123,29 +202,65 @@
           end
         end
 
+        it 'raises an ArgumentError when there is no button with that id' do
+          expect{ ShareProgress::Button.find(999999999)}.to raise_error ArgumentError
```

Comment:
It's a nitpick, but eventually there will be a button with that ID (I presume). Could we possibly do something like a negative number, or zero? Those should always be false, forever.